.k8/prater/ssv-node-prater-1-deployment.yml

```diff
@@ -0,0 +1,135 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: ssv-node-prater-1-svc
+  namespace: REPLACE_NAMESPACE
+  labels:
+    app: ssv-node-prater-1
+spec:
+  type: ClusterIP
+  ports:
+    - port: 12011
+      protocol: UDP
+      targetPort: 12011
+      name: port-12011
+    - port: 13011
+      protocol: TCP
+      targetPort: 13011
+      name: port-13011
+    - port: 15011
+      protocol: TCP
+      targetPort: 15011
+      name: port-15011
+  selector:
+    app: ssv-node-prater-1
+---
+apiVersion: REPLACE_API_VERSION
+kind: Deployment
+metadata:
+  labels:
+    app: ssv-node-prater-1
+  name: ssv-node-prater-1
+  namespace: REPLACE_NAMESPACE
+spec:
+  replicas: REPLACE_REPLICAS
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: ssv-node-prater-1
+  template:
+    metadata:
+      labels:
+        app: ssv-node-prater-1
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/role
+                    operator: In
+                    values:
+                      - ssv-main
+                  - key: beta.kubernetes.io/instance-type
+                    operator: In
+                    values:
+                      - m5a.4xlarge
+      containers:
+      - name: ssv-node-prater-1
+        image: REPLACE_DOCKER_REPO:REPLACE_IMAGETAG
+        imagePullPolicy: Always
+        resources:
+          limits:
+            cpu: REPLACE_NODES_CPU_LIMIT
+            memory: REPLACE_NODES_MEM_LIMIT
+        command: ["make", "start-node"]
+        ports:
+        - containerPort: 12011
+          name: port-12011
+          hostPort: 12011
+          protocol: UDP
+        - containerPort: 13011
+          name: port-13011
+          hostPort: 13011
+        - containerPort: 15011
+          name: port-15011
+          hostPort: 15011
+        env:
+          - name: SHARE_CONFIG
+            value: "./data1/share.yaml"
+          - name: BOOTNODES
+            valueFrom:
+              secretKeyRef:
+                name: config-secrets
+                key: boot_node_prater_enr
+          - name: LOG_LEVEL
+            value: "debug"
+          - name: DB_REPORTING
+            value: "false"
+          - name: PUBSUB_TRACE
+            value: "false"
+          - name: DISCOVERY_TYPE_KEY
+            value: "discv5"
+          - name: NETWORK
+            value: "jato-v2-stage"
```

Comment:
@moshe-blox 
If I understand right, its related to network config
If so we have to create a config for jato-v2-prater-prod